lib/ansible/parsing/yaml/objects.py

```diff
@@ -19,9 +19,11 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+import sys
 import yaml
 
-from ansible.module_utils.six import text_type
+from ansible import constants as C
```

Comment:
This creates a circular import, which can be seen in the following traceback in unit tests on Python 2.7:

```
02:24 ImportError while loading conftest '/root/ansible/test/units/plugins/become/conftest.py'.
02:24 test/units/plugins/become/conftest.py:12: in <module>
02:24     from ansible.cli.arguments import option_helpers as opt_help
02:24 lib/ansible/cli/__init__.py:18: in <module>
02:24     from ansible.cli.arguments import option_helpers as opt_help
02:24 lib/ansible/cli/arguments/option_helpers.py:17: in <module>
02:24     from ansible import constants as C
02:24 lib/ansible/constants.py:19: in <module>
02:24     from ansible.config.manager import ConfigManager, ensure_type, get_ini_config_value
02:24 lib/ansible/config/manager.py:32: in <module>
02:24     from ansible.parsing.yaml.objects import AnsibleVaultEncryptedUnicode
02:24 lib/ansible/parsing/yaml/objects.py:25: in <module>
02:24     from ansible import constants as C
02:24 E   ImportError: cannot import name constants
```

Reply:
I also don't see why the import is needed, it's not used by the changes.

Reply:
tis not, left over from previous iteration